File: C#/8-kyu/How good are you really?.cs
using System.Linq;

public class Kata
{
    public static bool BetterThanAverage(int[] ClassPoints, int YourPoints)
    {
        double classAverage = (double)ClassPoints.Sum() / ClassPoints.Length;
        
        return YourPoints > classAverage;
    }
}

File: TypeScript/8-kyu/How good are you really?.ts
export function betterThanAverage(classPoints: number[], yourPoints: number) : boolean {
  const totalPoints = classPoints.reduce((accum, score) => accum + score, 0);
  
  const average = (totalPoints + yourPoints) / (classPoints.length + 1);
  
  return yourPoints > average;
}
